support-frontend/assets/helpers/abTests/abtest.js

```diff
@@ -51,6 +52,11 @@ type Audiences = {
   [IsoCountry | CountryGroupId | 'ALL']: Audience
 };
 
+type AcquisitionABTest = {
```

Comment:
This type already exists in `acquisitions.js`

Reply:
Thanks Rupert! 👍